modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/AgentConfiguration.java

```diff
@@ -374,6 +374,12 @@ public String getAgentUpdateVersionUrlIfDefined() {
         return str;
     }
 
+    public long getAgentUpdateExitTimeout() {
+        long value = m_preferences.getLong(AgentConfigurationConstants.AGENT_UPDATE_EXIT_TIMEOUT_MSECS,
```

Comment:
I was just following the same notations that are along the file, should I just return right away then?